### ISSUE TITLE
[Feature Request]: New version of this mod for DQ3HD2D

### ISSUE THREAD

**Issue by agentoranger** [Lyall/P3RFix#52]
It seems like the community is using this mod for Dragon Quest III HD-2D, a humble request from someone that appreciates your work: can a version of this mod, or at least something similar be made for Dragon Quest III HD-2D?

Thanks in advance, I really appreciate your time.

Found in the wild:
https://www.youtube.com/watch?v=GtPOHgkndr4

**Comment by agentoranger** [comment]
Notably, I tested a few of the features that P3RFix provide on DQ3HD2D.  Tested injection P3RFix_v1.2.4_Steam
with SpecialK and dsound.dll.

Set all other options to false.

**[FPS Cap]** - FPS Cap works as expected and the FPS in game and in the menu will run at the provided FPS Cap.
AdjustFPSCap = true
Framerate = 144

**[Uncap 60FPS Menus]** - Does not appear to be needed.

;;;;;;;;;; Ultrawide/Narrower Fixes ;;;;;;;;;;

**[Fix HUD]** - Does not appear to be needed, the hud is generally unconstrained, enabling does not change hud appearance.
Enabled = false

**[Fix Aspect Ratio]** - Fix Aspect Ratio works as expected, correctly expands the game to fit ultra widescreen at 3440 x 1440.
Enabled = true

**[Fix FOV]** - Fixes FOV works as expected, correctly expands the FOV.
Enabled = true

**[Screen Percentage]** - Might work correctly, but this has interesting side effects introducing delays in NPC dialog.
Enabled = true
Value = 200

Known Issues:
When in ultra widescreen mode, damage indicators are left aligned, which misaligns damage test from the enemy that is being damaged.  The issue is not affected by [Fix HUD]

When in ultra widescreen mode, Speech and interaction indicators are misaligned in the same way that damage indicators are misaligned.

When in ultra widescreen mode, Texture and shadow pop in when viewed from areas of the screen that are not normally viewable.
